velox/core/QueryConfig.h

```diff
@@ -728,6 +735,12 @@ class QueryConfig {
     return get<int32_t>(kPrefixSortMinRows, 130);
   }
 
+  std::string sparkMapKeyDedupPolicy() const {
+    std::string policy = get<std::string>(kSparkMapKeyDedupPolicy, "EXCEPTION");
+    std::transform(res.begin(), res.end(), res.begin(), ::toupper);
```

Comment:
Is the policy must by "EXCEPTION" or "LAST_WIN", and shall we throw if it's not?